lib/uksignal/signal.c

```diff
@@ -360,7 +426,7 @@ int killpg(int pgrp, int sig)
 		return -1;
 	}
 
-	return kill(uk_syscall_r_getpid(), sig);
+	return kill(-pgrp, sig);
 }
 
 int raise(int sig)
```

Comment:
This definition will conflict with the one above if libc wrappers are enabled. I guess you could remove this one here.